libr/anal/dwarf_process.c

```diff
@@ -51,7 +51,7 @@ typedef struct dwarf_variable_t {
 } Variable;
 
 static void variable_free(Variable *var) {
-	free (var->name);
+	free ((char *) var->name);
```

Comment:
I think you should change `Variable.name` to `char *`. Right now `var->name` is set with things like `r_str_newf`, `strdup`, etc., so it is definitely not `const char *`.

Reply:
Well at first it was set by `const char *` from the raw dwarf information. And because there are 2 possibilitites of the name (name and linkage name) I wanted to not waste extra duplicating and freeing, so I've kept it `const char *` and strdup it at the end when I picked which of the names I want. I've refactored it to use separate variable for this and store the strduped name into the variable so I've removed the const.